core/app/models/spree/order_contents.rb

```diff
@@ -7,7 +7,9 @@ def initialize(order)
     end
 
     def add(variant, quantity = 1, options = {})
+      timestamp = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.